packages/vscode-extension/src/util/ExtensionUtil.ts

```diff
@@ -2,37 +2,33 @@
  ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
  ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
  */
-
 import * as vscode from 'vscode';
 
 // returns the root project folder of the active file in the editor
 // works fine with workspace with multiple project folders opened
-export function getRootProjectFolder(): string | null {
+export function getRootProjectFolder(): string | undefined {
```

Comment:
What was the reason to change this from null to undefined?

Reply:
that later on, when we call this method and assign it's result to any variable, we can use the `?` parameter:
```
protected executionPath?: string;
[...]
this.executionPath = getRootProjectFolder();
```
(this code is from BaseAction.ts)